src/character/mod.rs

```diff
@@ -23,6 +30,7 @@ pub struct Character {
 
     pub strength: i32,
     pub speed: i32,
+    pub condition: Option<Condition>,
```

Comment:
instead of having `Option<Condition>` wouldn't it work to have a "Normal" enum variant?

Reply:
yes I think so. Will give it a try